### PR TITLE
Classifies Security's Toilet as a toilet rather than a Vacant Office

### PR DESCRIPTION
**Pull request by Ty-Omaha**
This caused wizards who selected "vacant office" to teleport to, ending up having a chance to teleport to security's toilets. Or anomalys that say they're at the "vacant office" but are really in sec toilets.

:cl:
fix: Security's Toilet is no longer classified as a Vacant Office
/:cl:

### CODE REVIEW

File: code/game/area/Space Station 13 areas.dm [hunk 1]
```diff
@@ -1795,14 +1803,6 @@ area/security/podbay
 	name = "Security Post - Science"
 	icon_state = "checkpoint1"
 
-/area/security/vacantoffice
-	name = "\improper Vacant Office"
-	icon_state = "security"
-
-/area/security/vacantoffice2
```
Comment (SamHPurp) [inline code/game/area/Space Station 13 areas.dm]: tbh may as well leave these in.

Comment (SamHPurp) [reply code/game/area/Space Station 13 areas.dm]: 👍 otherwise

**Comment by LightFire53** [comment]
I knew there was something off when I kept teleporting to sec as a wizard...

**Comment by FreeStylaLT** [comment]
😅

File: code/game/area/Space Station 13 areas.dm (hunk 1 above)
Comment (Ty-Omaha) [reply code/game/area/Space Station 13 areas.dm]: eh I'd rather keep it removed so in the future if someone needs to make a vacant office area there isn't confusion about which one to use

**Comment by Ty-Omaha** [comment]
Git doesn't want to update anything for whatever reason; so I can't fix the conflicts

**Pull request closed without merging**